assets/js/blocks/cart-checkout/payment-methods/style.scss

```diff
@@ -235,6 +235,10 @@
 
 .wc-block-components-radio-control-accordion-content {
 	padding: 0 $gap em($gap) $gap;
+
+	&:empty {
```

Comment:
this does not work for me because the div has an empty div inside and it's not empty.
But if I move the styles to the child div it works as expected.

```
.wc-block-components-radio-control-accordion-content {
	div {
		padding: 0 $gap em($gap) $gap;

		&:empty {
			display: none;
		}
	}
}
```

Reply:
let me know if it works for you